uber/templates/preregistration/preregbase.html

```diff
@@ -67,37 +67,56 @@
             selector: '.badge-type-selector',
             options: [{
                 title: '{% if c.PAGE_PATH == "/preregistration/form" or c.PAGE_PATH == "/preregistration/dealer_registration" %}Attending{% else %}{{ attendee.ribbon_and_or_badge }}{% endif %}',
-                description: 'Allows access to the convention for its duration. {% price_notice "Preregistration" c.PREREG_TAKEDOWN %}',
+                description: 'Allows access to the convention for its duration.',
                 extra: 0
             }]
         };
         {% if c.SHIRT_LEVEL in c.PREREG_DONATION_TIERS and not attendee.gets_free_shirt %}
             BADGE_TYPES.options.push({
                 title: 'Add a tshirt',
-                description: 'Add a {{ c.EVENT_NAME }} themed t-shirt to your registration. {% price_notice "Preregistration" c.PREREG_TAKEDOWN c.SHIRT_LEVEL %}',
+                description: 'Add a {{ c.EVENT_NAME }} themed t-shirt to your registration.',
                 extra: {{ c.SHIRT_LEVEL }}
             });
         {% endif %}
         {% if c.SUPPORTER_LEVEL in c.PREREG_DONATION_TIERS and c.SUPPORTER_AVAILABLE %}
             BADGE_TYPES.options.push({
                 title: 'Supporter',
-                description: 'Donate extra and get more swag with your registration. {% price_notice "Supporter registration" c.SUPPORTER_DEADLINE c.SUPPORTER_LEVEL %}',
+                description: 'Donate extra and get more swag with your registration.',
                 extra: {{ c.SUPPORTER_LEVEL }}
             });
         {% endif %}
 
         var togglePrices = function () {
-            var showTotalPrices = {% if c.PAGE_PATH == '/preregistration/form' %}$.val('badge_type') === {{ c.ATTENDEE_BADGE }}{% else %}false{% endif %};
+            var showTotalPrices = {% if c.PAGE_PATH == '/preregistration/form' and not c.DONATION_SLIDER_DISPLAY %}($.val('badge_type') === {{ c.ATTENDEE_BADGE }}){% else %}false{% endif %};
             $.each(BADGE_TYPES.options, function (i, type) {
                 var $price = $(BADGE_TYPES.selector).slice(i, i + 1).find('.price').empty();
+                var $price_notice = $(BADGE_TYPES.selector).slice(i, i + 1).find('.price_notice').empty();
```

Comment:
Minor style note: since we're not repeating ``$(BADGE_TYPES.selector).slice(i, i + 1)`` we should probably makethat its own variable so as to not need the copy/paste.  Or not, since this a fairly small amount of repeating, so it might be within our tolerance for repeated code.